README.md

```diff
@@ -431,6 +431,18 @@ new Foo() instanceof null;
 
 This is not a part of the specification. It's just a bug that has now been fixed, so there shouldn't be a problem with it in the future.
 
+### Super constructor null of Foo is not a constructor
+It's continuation of story with previous bug in modern environment (tested with Chrome 71 and nodejs v11.8.0) 
+```js
+class Foo extends null {}
+new Foo() instanceof null;
+// > TypeError: Super constructor null of Foo is not a constructor
+// >     at … … …
+```
+
+### 💡 Explanation: 
+TBD
```

Comment:
Please, make a research and add the explanation

Reply:
Because:

```js
console.dir(Object.getPrototypeOf(Foo.prototype)); // is null
```

if the class has no constructor the call from prototype chain. But in the parent has no constructor. Just in case, I’ll clarify that null is an object. Therefore, you can inherit from it (although in the world of the OOP for such terms would have beaten me).

So you can't call the null constructor.

If you change this code:
```js
class Foo extends null {
    constructor() {
        console.log(111);
    }
}
```

You se the error:
```
ReferenceError: Must call super constructor in derived class before accessing 'this' or returning from derived constructor
```

And if add super:
```js
class Foo extends null {
    constructor() {
        console.log(111);
        super();
    }
}
```

JS throw error:
```
TypeError: Super constructor null of Foo is not a constructor
```

This is my explanation based on my experience and understanding how the  JS works.

Reply:
@linegel added a nice explanation. Please, add it to your PR.